packages/workspace/src/generators/workspace/files/README.md__tmpl__

```diff
@@ -12,9 +12,9 @@ This project was generated using [Nx](https://nx.dev).
 
 [Nx Documentation](https://nx.dev/angular)
 
-[10-minute video showing all Nx features](https://nx.dev/angular/getting-started/what-is-nx)
+[10-minute video showing all Nx features](https://nx.dev/latest/angular/getting-started/intro)
```

Comment:
Same here https://nx.dev/getting-started/intro

Reply:
I've changed it for the both links